content/sensu-go/5.20/operations/deploy-sensu/cluster-sensu.md

```diff
@@ -262,11 +262,44 @@ See [Secure Sensu][16] for information about cluster security.
 
 ## Use an external etcd cluster
 
-To use Sensu with an external etcd cluster, you must have etcd 3.3.2 or newer.
-To stand up an external etcd cluster, follow etcd's [clustering guide][2] using the same store configuration.
+Using Sensu with an external etcd cluster requires etcd 3.3.2 or newer but is not compatible with etcd 3.4.0 or newer.
```

Comment:
Sensu is compatible with Etcd 3.4.0 or greater however we've seen a ~50% decrease in performance when using Etcd-only configurations. We may want to specify that it's compatible but slower than Etcd 3.3.

Reply:
This statement makes so many things make sense now.

Reply:
I really REALLY wish the 3.4.x performance callout had been present when we migrated from sensu-core in 4 environments. :(

Reply:
I corrected this in https://github.com/sensu/sensu-docs/pull/3019. Thank you @amdprophet